Add metadata and ontologies informations (#104)
NFDI4Microbiota/nfdi4microbiota-knowledge-base: docs/_Research-Data-Management/03-md.md

layout: default
docs_css: markdown
redirect_from: /Research-Data-Management
empty: false
hide: false
---

# Metadata

> ## Metadata is data about data

Reviewer: ```suggestion
## Metadata describes your data
```
or ## Metadata is information about your data 
personally I like the heading but maybe to much data

Before we delve into specifications on what Metadata standards for the Microbiology community are or would be, let us explain what metadata is. 

> Metadata is information that describes other information in order to help you understand or use it. In the simplest terms, metadata is **data** about data. So, in general, metadata provides us with information about other data but does not tell us anything about the content of the data itself.

Reviewer: ```suggestion
In general, metadata provides us with information about other data but does not tell us anything about the content of the data itself. Instead, it describes other types of information to help you understand or use the data you are working with. In the simplest terms, metadata is **data** about data.
```

If you find our explanation lacking, we refer you to the [FAIR Cook Book](https://faircookbook.elixir-europe.org/content/home.html), more specifically to the [FAIR and the notion of metadata](https://faircookbook.elixir-europe.org/content/recipes/introduction/metadata-fair.html) section.

## When should you collect your metadata

> As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this KnowledgeBase: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.

Reviewer: ```suggestion
As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this Knowledge Base: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.
```
"Knowledge Base" as on the N4M webpage

> Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will one year older me understand what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 

Reviewer: ```suggestion
Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will I still understand in a year what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 
```

> These and other considerations not noted here should be thoroughly thought out before the start of experimental procedures. Some of the metadata can even be collected and documented before the start of experiments if you already know how to collect your samples, sequence them (if sequencing is a part of the analysis), and analyze them. 

Reviewer: ```suggestion
These and other considerations should be thoroughly thought out before the start of your experimental procedures. Some of the metadata can even be collected and documented before starting the experiments if you already know how to collect your samples, process, sequence them (if sequencing is a part of the analysis), and analyze them. 
```

layout: default
docs_css: markdown
redirect_from: /Research-Data-Management
empty: false
hide: false
---

# Metadata

## Metadata is data about data

> Before we delve into specifications on what Metadata standards for the Microbiology community are or would be, let us explain what metadata is. 

Reviewer 2: ```suggestion
Before we delve into specifications on what metadata standards for the microbiology community are, let us explain what metadata is. 
```

Not a huge fan of the would be formulation

In general, metadata provides us with information about other data but does not tell us anything about the content of the data itself. Instead, it describes other types of information to help you understand or use the data you are working with. In the simplest terms, metadata is **data** about data.

> If you find our explanation lacking, we refer you to the [FAIR Cook Book](https://faircookbook.elixir-europe.org/content/home.html), more specifically to the [FAIR and the notion of metadata](https://faircookbook.elixir-europe.org/content/recipes/introduction/metadata-fair.html) section.

Reviewer 2: ```suggestion
For more details on the distinction between different types of metadata,  we refer you to the FAIR Cookbook recipe [FAIR and the notion of metadata](https://w3id.org/faircookbook/FCB068) section.
```

- removed the formulation that shoot ourserveles in the foot
- use the persistent identifier for the recipe

## When should you collect your metadata

> As is usually the case in sciences, your research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this KnowledgeBase: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.

Reviewer 2: ```suggestion
As expected, your scientific research (and the wider microbiological community) can benefit highly from the rigorous and timely planning of your experiments, including metadata collection. In this case, we refer you to other subsections of this KnowledgeBase: [**Data Management Plans (DMPs)**](./08-dmp.md) that could help you plan your experiments.
```

- feels like stating the obvious, so I just tone it down

Metadata collection should be planned, but at the same time, it can be overwhelming. What amount of metadata is enough to describe your data? Was it crucial to note down the pipette I was using, or should I have noted down the location of the sampling site? Will one year older me understand what I wrote down in my notebook/ELN? Will other researchers make sense of the (meta)data I collected? Will other researchers be able to replicate my research if I did not note down my in-house DNA extraction protocol? 

These and other considerations not noted here should be thoroughly thought out before the start of experimental procedures. Some of the metadata can even be collected and documented before the start of experiments if you already know how to collect your samples, sequence them (if sequencing is a part of the analysis), and analyze them. 

## Metadata collection example

> We will look into an example of microbiological environmental metadata, where we gather samples from a forest environment, specifically plant rhizosphere, and we will be doing amplicon and metagenomic sequencing. We will not dive specifically into all omics types and biological/environmental on this page. Instead, we encourage readers to read our [MetadataStandards](https://github.com/NFDI4Microbiota/MetadataStandards/tree/main) resource repository. 

Reviewer 2: ```suggestion
We will look into an example of microbiological environmental metadata, where we gather samples from a forest environment, specifically plant rhizosphere, and we will be doing amplicon and metagenomic sequencing. We will not dive specifically into all omics types and biological/environmental on this page. Instead, we encourage readers to read our [MetadataStandards](https://github.com/NFDI4Microbiota/MetadataStandards) resource repository. 
```

overly specific URL, works only if you're citing a specific version

layout: default
docs_css: markdown
redirect_from: /Research-Data-Management
empty: false
hide: false
---

# Metadata

## What is metadata? 

> Before we delve into specifications on what Metadata standards for the Microbiology community are or would be, let us explain what metadata is and how you have probably already used it at least once. 

Reviewer: ```suggestion
Before we delve into specifications on what metadata standards for the microbiology community are, let us explain what metadata is and how to use it. Maybe you are already doing it unintentionally.
```

> Metadata is information that describes other information in order to help you understand or use it. In the simplest terms, metadata is **data** about data. So, in general, metadata provides us with information about other data but does not tell us anything about the content of the data itself. 

Reviewer: ```suggestion
In general, metadata provides us with information about other data but does not tell us anything about the content of the data itself. Instead, it describes other type of information to help you understand or use the data you are working with. In the simplest terms, metadata is **data** about data.
```

> For example, when you use your smartphone camera to take photos of your loved one, your device automatically creates the corresponding metadata regarding that photograph. The metadata, in this case, should at least contain the following (depending on your device settings):

Reviewer: ```suggestion
For example, when you use your smartphone camera to take photos of your loved one, your device automatically creates the corresponding metadata regarding that photograph. In this case, the metadata should at least contain the following (depending on your device settings):
```
